src/include/platform/internal/GenericConnectivityManagerImpl_UDP.ipp

```diff
@@ -33,8 +34,8 @@ namespace Internal {
 template <class ImplClass>
 chip::Inet::EndPointManager<Inet::UDPEndPoint> & GenericConnectivityManagerImpl_UDP<ImplClass>::_UDPEndPointManager()
 {
-    static chip::Inet::UDPEndPointManagerImpl sUDPEndPointManagerImpl;
-    return sUDPEndPointManagerImpl;
+    static Global<chip::Inet::UDPEndPointManagerImpl> sUDPEndPointManagerImpl;
```

Comment:
Please move the Global out of the method, otherwise we end up with the compiler-generated lazy init + whatever Global does.